### DISCUSSION TITLE
Share tips and tricks working with Lua

### DISCUSSION THREAD

**Discussion by max-baz** [sayanarijit/xplr#250] [Show and tell]
Because we are all learning, I thought it would be nice to share tips on how you work with Lua to organize your configs.

I tried [StyLua](https://github.com/JohnnyMorganz/StyLua) and am very happy with its results, the only modification I made is to prefer spaces over tabs 🙊 

I also tried [lua-language-server](https://github.com/sumneko/lua-language-server) but didn't really get any benefit of it.

**Comment by sayanarijit** [comment]
I learned a weird trick... While writing a lua plugin, if you place [this file](https://github.com/sayanarijit/xplr/blob/main/src/init.lua) inside the project directory, you can get lsp hints.
